### ISSUE TITLE
Implement Hangouts Chat item

### ISSUE THREAD

**Issue by TortoiseWrath** [multacc/multacc#78]

**Comment by mayank99** [comment]
I don't think this is possible. The "profile" is just a Gmail account, and their Chat API only allows bots

**Comment by TortoiseWrath** [comment]
the problem is that Hangouts Chat is impossible to google since all you get is Hangouts chat which is deprecated

Hangouts chat is dead, long live Hangouts Chat
